camel-k-runtime-core/src/test/java/org/apache/camel/k/support/RuntimeSupportTest.java

```diff
@@ -16,21 +16,20 @@
  */
 package org.apache.camel.k.support;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 import java.util.List;
 import java.util.Properties;
 
 import org.apache.camel.CamelContext;
 import org.apache.camel.Ordered;
 import org.apache.camel.component.properties.PropertiesComponent;
 import org.apache.camel.impl.DefaultCamelContext;
-import org.apache.camel.impl.ExplicitCamelContextNameStrategy;
+import org.apache.camel.impl.engine.ExplicitCamelContextNameStrategy;
```

Comment:
If you need this one for being able to set the name easily, then the setName api is now public. But I think this may be from M4 onwards

Reply:
yup, will clean up once M4 will be out out